src/plugins/kibana_overview/public/components/overview/overview.tsx

```diff
@@ -171,12 +181,21 @@ export const Overview: FC<Props> = ({ newsFetchResult, solutions, features }) =>
   }
 
   if (isNewKibanaInstance) {
+    const hasCustomBranding = customBranding.hasCustomBranding$.pipe(() => {
```

Comment:
We should be subscribing to the observable in our component, not in the consuming plugins.